src/main/groovy/life/qbic/datamodel/dtos/imaging/parameters/lightmicroscopy/LightMicroscopyParameters.groovy

```diff
@@ -1,12 +1,107 @@
 package life.qbic.datamodel.dtos.imaging.parameters.lightmicroscopy
 
+import life.qbic.datamodel.dtos.imaging.parameters.AcquisitionParameters
+import life.qbic.datamodel.dtos.imaging.properties.Area
+import life.qbic.datamodel.dtos.imaging.properties.Position
+import life.qbic.datamodel.dtos.imaging.properties.Power
+import life.qbic.datamodel.dtos.imaging.properties.Resolution
+import life.qbic.datamodel.dtos.imaging.properties.Time
+import life.qbic.datamodel.dtos.imaging.properties.Unit
+import life.qbic.datamodel.dtos.imaging.properties.WaveLength
+
 /**
- * <class short description - 1 Line!>
+ * Description of light microscope parameters during image analysis.
+ *
+ * This class serves as DTO and contains information about
+ * general light microscope parameters during an image analysis.
  *
- * <More detailed description - When to use, what it solves, etc.>
+ * This class shall be extended, if you need to add more technology-specific
+ * properties.
  *
  * @author Sven Fillinger
- * @since <versiontag>
+ * @since 1.10
  */
-class LightMicroscopyParameters {
+class LightMicroscopyParameters extends AcquisitionParameters{
+
+    /**
+     * It is light that excites fluorochromes. Its the light that gets absorbed by the specimen
+     * A sufficient intensity needs to match the excitation range of the fluorochrome in order to get a signal.
+     *
+     * Measured as light in wavelength per channel
+     */
+    private final List<WaveLength> excitationPerChannel
+
+    /**
+     * Wavelength of emission light per channel. Its the light that passes the specimen and is not absorbed.
+     *
+     * Measured as light in wavelength per channel
+     */
+    private final List<WaveLength> emissionPerChannel
+
+    /**
+     * Total Internal Reflection Fluorescence (TIRF)
+     * Amount of power of the laser beam of the TIRF lens
+     *
+     * Power is measured in milliwats (mW)
+     */
+    private final Power laserPowerTIRFLens
+
+    /**
+     * Amount of power of the laser beam split by the beam splitter.
+     * A beam splitter is an optical device that splits the beam of light
+     *
+     * Power is measured in milliwats (mW)
+     */
+    private final Power laserPowerBeamSplitter
+
+    /**
+     * Time required to generate an image with the camera
+     *
+     * In the range of femtoseconds to picoseconds
+     */
+    private final Time cameraAcquisitionTime
```

Comment:
Maybe we go for the Java-native `Date` object here
```suggestion
    private final Time cameraAcquisitionTime
```

Reply:
Good Idea, a accepted your suggestion. For me a fixed unit would be fine. Maybe we need to check with @luiskuhn to make sure this is a good unit to choose?